psi-sample/README.md

```diff
@@ -51,72 +51,98 @@ Merchant - Merchant passes credit card numbers of all users who have made a purc
-Enclave code signer: 4502FEF2B5973A9DCF2F5C85358ED9F099C7738300364A7D7451371E43694A85
-```
+Add conclave-client dependency to the client build.gradle. This consists of the EnclaveClient class which will be used 
+to encrypt and send mails to enclave
 
-#### A note about enclave constraint
-In this sample the `code signer` is used as enclave constraint, but you can also use the `code hash`. If you want to use it, remember to change the code of the client to:
+    implementation "com.r3.conclave:conclave-web-client:$conclaveVersion"
 
-`EnclaveConstraint.parse("C:"+ constraint +" SEC:INSECURE" ).check(attestation);`
+We will also add picocli dependency to the client, thus can be used by clients to provide arguments via command line
 
-Read more in the [documentation](https://docs.conclave.net/enclave-configuration.html#productid).
+    implementation "info.picocli:picocli:4.6.1"
 
-## How to run in mock mode
+Similar to host add shadowJar plugin to client as well
+
+Use shadowJar plugin to build the client jar
+
+    ./gradlew client:shadowJar
+You can also run both together like this
 
-We don't need to use the container-gradle plugin to load an enclave in mock mode.
+    ./gradlew host:shadowJar client:shadowJar
 
-Start the host on a non-Linux system, which will build the enclave and host:
+Use below command to provide command line arguments to client
+Start the merchant client
 
-    ./gradlew -PenclaveMode=mock host:run
+     java -jar client/build/libs/client.jar  --constraint "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" --url "http://localhost:8080" MERCHANT 88
 
-On your terminal, once the host starts, pass in the credit card numbers via Merchant terminal
+Start the service provider client
 
-    ./gradlew client:run --args="MERCHANT <CONSTRAINT> 1123 4456 777998 88988898"
+    java -jar client/build/libs/client.jar  --constraint "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" --url "http://localhost:8080" SERVICE-PROVIDER 88
 
-On your terminal, start Service Provider and pass in the credit card numbers of users who have clicked the ad
+Once you run both the clients, the enclave calculates the ad conversion rate and sends it to both the clients
 
-    ./gradlew client:run --args="SERVICE-PROVIDER <CONSTRAINT> 88988898 77879 00989"
+    Ad Conversion Rate is : 100.0
 
-Once both the clients pass in the credit card numbers, the host calculates the ad conversion rate within the enclave and sends it to both the clients.
-For mock mode specify 0000000000000000000000000000000000000000000000000000000000000000 for the <CONSTRAINT> parameter.
+
+#### A note about enclave constraint
+In this sample the `code signer` is used as enclave constraint, but you can also use the `code hash`. 
+If you want to use it, remember to change the code of the client to:
+
+`EnclaveConstraint.parse("C:"+ constraint +" SEC:INSECURE" ).check(attestation);`
```

Comment:
You don't change the code, but rather simply pass in a different arg to the `--constraint` flag.

Reply:
done